nest/05.cheat_sheet.md

```diff
@@ -0,0 +1,18 @@
+# 5 Cheat sheet
+In this chapter you will find a cheat sheet, detailing all the commands and options available for `nest` and `finest`, with a short explanation on what they do, and how to use them.
```

Comment:
```suggestion
In this chapter you will find a cheat sheet, detailing all the commands and options available for `nest` and `finest`, with a brief explanation on what they do, and how to use them.
```
(no difference in meaning, I just prefer brief)

Reply:
Could you rephrase the first sentence of your comment? :grimacing:

Reply:
I meant that I prefer "brief" over "short", but that there is no difference in meaning between both versions.

Reply:
I meant from your other comment. The one just above starting with `Stupid idea`, it's like words are missing or something

Reply:
damn, apparently markdown swallows words in <>
Going to edit that.